nav.js

```diff
@@ -0,0 +1,6 @@
+const toggleButton = document.getElementsByClassName('toggle-button')[0];
```

Comment:
Added javascript to allow user to see nav links when burger menu icon is clicked and close it when clicked again after